index.html

```diff
@@ -939,17 +961,15 @@ <h3><dfn>disconnect()</dfn> method</h3>
                 [=this=] using the [=smart card task source=] which performs the
                 following steps:
                 <ol>
-                  <li>Set
-                    [=this=].{{SmartCardConnection/[[context]]}}.{{SmartCardContext/[[operationInProgress]]}}
-                    to `false`.</li>
+                  <li>[=Clear the operationInProgress=] of
+                    [=this=].{{SmartCardConnection/[[context]]}}.</li>
                   <li>If |responseCode| is not `SCARD_S_SUCCESS`, [=reject=]
                     |promise| with a {{DOMException}}
                     {{SmartCardError/corresponding}} to |responseCode| and abort
                     these steps.</li>
                   <li>Destroy [=this=].{{SmartCardConnection/[[comm]]}}.</li>
                   <li>Set [=this=].{{SmartCardConnection/[[comm]]}} to `null`.</li>
```

Comment:
Also remove this connection from the set of active connections.

Reply:
I think this adds complexity (more code) for arguable value (minor, probably undetectable, optimisation). The overall logic still works pretty much the same without this. "end any pending transaction" already bails out if there's no transaction.

A PC/SC connection might become invalid even without `SmartCardConnection.disconnect()` being called:
- when a card is removed from the reader (unless it's a direct connection).
- when the reader itself is physically disconnected from the system.
- when the PC/SC resource manager service shuts down, crashes or simply terminates from some reason.
 
Meaning that one can never guarantee that a connection is **actually** active.

Reply:
I still think it's worth at least making the state management within the given algorithms consistent.

Reply:
After sitting with this for awhile I think maybe this is okay,